DocumentDataSubmission/src/main/resources/wsdl/AdapterComponentDDSSecured.wsdl

```diff
@@ -34,32 +34,32 @@
         <documentation>Register Document Set Response</documentation>
         <part name="body" element="rs:RegistryResponse"/>
     </message>
-    <portType name="AdapterComponentXDRSecured_PortType">
+    <portType name="AdapterComponentDDSSecured_PortType">
         <operation name="RegisterDocumentSetb">
-            <input message="tns:RegisterDocumentSet-b_Message" wsaw:Action="urn:gov:hhs:fha:nhinc:adaptercomponentxdrsecured:data:RegisterDocumentSet-b"/>
-            <output message="tns:RegisterDocumentSet-bResponse_Message" wsaw:Action="urn:gov:hhs:fha:nhinc:adaptercomponentxdrsecured:data:RegisterDocumentSet-bResponse"/>
+            <input message="tns:RegisterDocumentSet-b_Message" wsaw:Action="urn:gov:hhs:fha:nhinc:adaptercomponentddssecured:data:RegisterDocumentSet-b"/>
+            <output message="tns:RegisterDocumentSet-bResponse_Message" wsaw:Action="urn:gov:hhs:fha:nhinc:adaptercomponentsecured:data:RegisterDocumentSet-bResponse"/>
```

Comment:
wsaw:action should be urn:gov:hhs:fha:nhinc:adaptercomponentddssecured:data:RegisterDocumentSet-bResponse